src/engine/archive.h

```diff
@@ -38,9 +40,9 @@ namespace CityFlow {
         };
 
         VehiclePool vehiclePool;
-        std::map<const Drivable *, DrivableArchive> drivableArchive;
-        std::map<const Flow *, FlowArchive> flowArchive;
-        std::map<const TrafficLight *, TrafficLightArchive> trafficLightArchive;
+        std::map<const Drivable *, DrivableArchive> drivablesArchive;
+        std::map<const Flow *, FlowArchive> flowsArchive;
+        std::map<const Intersection *, TrafficLightArchive> trafficLightsArchive;
```

Comment:
I don't agree with these spelling changes

Reply:
It is neccessary to distinguish a certain DrivableArchive object and the map object. The old name 'drivableArchive' may be confusing, especially in some local scope where we need to specify a certain DrivableArchive object. Maybe there are better names?